AdobeStockClient/Model/Client.php

```diff
@@ -235,9 +235,15 @@ public function getFullEntitlementQuota(): UserQuotaInterface
     /**
      * @inheritdoc
      */
-    public function getQuotaConfirmationMessage(int $contentId): string
+    public function getQuotaInformation(int $contentId): array
     {
-        return $this->getLicenseInfo($contentId)->getPurchaseOptions()->getMessage();
+        $licenseInfo = $this->getLicenseInfo($contentId);
+        $quota = $licenseInfo->getEntitlement()->getQuota();
+
+        return [
+            'message' => $licenseInfo->getPurchaseOptions()->getMessage(),
```

Comment:
It would be good to introduce a `QuotaInterface` for quota and I think we can merge the `Client::getQuotaInformation` method with `ClientInterface::getFullEntitlementQuota` and `ClientInterface::getQuota ` having that data interface in place

Reply:
will be done in https://github.com/magento/adobe-stock-integration/pull/459